docs/how_to_use/how_to_build.md

```diff
@@ -182,16 +182,29 @@ http_archive(
 
 ## Debian packaging
 
+There are two ways to install the Tachyon package. While it is recommended to download prebuilt binaries, it is also possible to build the package from source if necessary.
+
+### Install package from pre-built binaries
+
+``` shell
```

Comment:
Maybe adding reasons for each section might help the user out with understanding what they're doing and that they are supposed to run the following commands. Perhaps take inspiration from [this](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#prepare-fedora)? (same for building from source section)